Profile/EducationProfile.cs

```diff
@@ -0,0 +1,62 @@
+﻿using MarsQA_1.Helpers;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System.Threading;
+
+namespace MarsQA_1.SpecflowPages.Pages
+{
+    public class EducationProfile

```

Comment:
Minor thing.

You could move this class and the other classes into a class called Profile.cs and group them in regions. Because you are testing the UI in the Profile page. Imagine having to create 100 classes for every single feature and needing to create a 100 more instances.

Do you agree?

Reply:
Yes I agree. But as a beginner I think it would be confusing to me to have all webelements declared in one place and writing all the tests in the same place. I would rather keep them in different classes for better understanding and also I could save some time as well. I would definitely keep this in mind and consider your advice while working on the competition task. Hope you understand.